languages/rust/oso/src/oso.rs

```diff
@@ -241,6 +261,46 @@ impl Oso {
         );
         Ok(())
     }
+
+    pub fn enable_roles(&mut self) -> crate::Result<()> {
+        if self.polar_roles_enabled {
+            return Ok(());
+        }
+
+        self.inner.enable_roles()?;
+
+        if !self.host.has_class(OSO_INTERNAL_ROLES_HELPER) {
+            self.register_class(
```

Comment:
It would be simpler to just include this with the built-in classes and register it when the host is initialized, before `enable_roles`. Is there any reason why we might not want to do that?

Reply:
I went ahead & did it but let me know if there's a problem!

Reply:
That seems fine to me.